# file: C:\Users\Asus\AppData\Roaming\Python\Python39\site-packages\exceptiongroup\_exceptions.py
# hypothesis_version: 6.141.1

['BaseExceptionGroup', 'ExceptionGroup', '_BaseExceptionT', '_BaseExceptionT_co', '_ExceptionGroupSelf', '_ExceptionT', '_ExceptionT_co', '__notes__', 's']